[Docs] Init a cookbook (#128)
Sylius/Stack: docs/cookbook/admin_panel/index.md

# How to customize your admin panel

## How to customize the sidebar logo

> To customize the sidebar logo, you need to configure the new template to use using the following configuration.

Reviewer: `to use using` -- English not englishing. :slightly_smiling_face: 
 
for example,
```suggestion
To customize the sidebar logo, you need to set new logo template at `sylius_admin.common.component.sidebar.logo` twig hook.
```

```yaml
# config/packages/sylius_bootstrap_admin_ui.yaml
# ...
sylius_twig_hooks:
    hooks:
        # ...
        'sylius_admin.common.component.sidebar.logo':
            image:
                # template: '@SyliusBootstrapAdminUi/shared/crud/common/sidebar/logo/image.html.twig'
                template: 'shared/crud/common/sidebar/logo/image.html.twig'

```

```php
// config/packages/sylius_bootstrap_admin_ui.php
use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;

return static function (ContainerConfigurator $containerConfigurator): void {
    // ...

    // Add these following lines to define your own Twig template for the logo.
    $containerConfigurator->extension('sylius_twig_hooks', [
        'hooks' => [
            'sylius_admin.common.component.sidebar.logo' => [
                'image' => [
                    // 'template' => '@SyliusBootstrapAdminUi/shared/crud/common/sidebar/logo/image.html.twig',
                    'template' => 'shared/crud/common/sidebar/logo/image.html.twig',
                ],
            ],
        ],
        
    ]);
};

```

```twig
{# templates/shared/crud/common/sidebar/logo/image.html.twig #}

> <img src="{{ asset('images/logo.png') }}" alt="CFC" class="navbar-brand-image" />

Reviewer 2: ```suggestion
<img src="{{ asset('images/logo.png') }}" alt="Your Brand name" class="navbar-brand-image" />
```

IMO, it's nore suitable ;)

  
```

## How to customize the login page logo

> To customize the login page logo, you need to configure the new template to use using the following configuration.

Reviewer: Same

> To customize the login page logo,you need to set new logo template at `sylius_admin.security.login.logo` twig hook.

Reviewer: ```suggestion
To customize the login page logo, you need to set new logo template at `sylius_admin.security.login.logo` twig hook.
```
